protocol/x/clob/keeper/process_operations.go

```diff
@@ -675,6 +675,34 @@ func (k Keeper) PersistMatchDeleveragingToState(
 	}
 	deltaBaseQuantumsIsNegative := position.GetIsLong()
 
+	// If there are zero-fill deleveraging operations, this is a sentinel value to indicate a subaccount could not be
+	// liquidated or deleveraged and still has negative equity. Mark the current block number in state to indicate a
+	// negative TNC subaccount was seen.
+	if len(matchDeleveraging.GetFills()) == 0 {
```

Comment:
for my understanding, what happens when a -ve TNC subaccount's position is only partially deleveraged? do we have a normal deleveraging operation with len(fills) > 0 followed by a empty deleveraging operation?

Reply:
+1 on question, also good to have a test for this case if true

Reply:
Answered synchronously, but yeah